### DISCUSSION TITLE
Examples compile but put my esp32 keeps resetting on boot

### DISCUSSION THREAD

**Discussion by AAfetisov** [ukw100/IRMP#97] [Q&A]
device: esp32 wroom 32D
examples: AllProtocols
This looping on serial monitor:
```
ets Jul 29 2019 12:21:46

rst:0x10 (RTCWDT_RTC_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:1
load:0x3fff0030,len:1344
load:0x40078000,len:13964
load:0x40080400,len:3600
entry 0x400805f0
```

I tried other examples involving receiver with similar results. Wokwi examles don't work
https://wokwi.com/projects/298945438795432456

tried connecting to external power source and monitoring via uart. same results

my arduino board settings just in case
![2024-08-06-00 01@2x](https://github.com/user-attachments/assets/1958a59e-ae1c-466a-aacf-e802943405fc)

**Comment by ArminJo** [comment]
it compiles and run for me with USE_NO_LCD enabled and ESP32 Core 3.0.2
